lib/runtime/life/resolver.go

```diff
@@ -513,6 +573,757 @@ func ext_offchain_index_set_version_1(vm *exec.VirtualMachine) int64 {
 	return 0
 }
 
+func ext_default_child_storage_set_version_1(vm *exec.VirtualMachine) int64 {
+	logger.Trace("[ext_default_child_storage_set_version_1] executing...")
+	storage := ctx.Storage
+	memory := vm.Memory
+
+	childStorageKeySpan := vm.GetCurrentFrame().Locals[0]
+	childStorageKey := asMemorySlice(memory, childStorageKeySpan)
+	keySpan := vm.GetCurrentFrame().Locals[1]
+	key := asMemorySlice(memory, keySpan)
+	valueSpan := vm.GetCurrentFrame().Locals[2]
+	value := asMemorySlice(memory, valueSpan)
+
+	cp := make([]byte, len(value))
+	copy(cp, value)
+
+	err := storage.SetChildStorage(childStorageKey, key, cp)
+	if err != nil {
+		logger.Error("[ext_default_child_storage_set_version_1] failed to set value in child storage", "error", err)
+		return 0
+	}
+	// todo(ed) what is this supposed to return?
+	return 0
```

Comment:
life forces the function to have a return value, even if it's unused, right? I think in that case, you can return anything, since it's unused anyways

Reply:
That makes sense.